l3_dump.py

```diff
@@ -38,7 +87,11 @@
         row = file.read(32)
         tid, loc, ptr, arg1, arg2 = struct.unpack('<iiQQQ', row)
         offs = ptr - fibase - rodata_offs
+
         if loc == 0:
             print(f"{tid=} '{strings[offs]}' {arg1=} {arg2=}")
-        else:
+        elif DECODE_LOC_ID == 0:
             print(f"{tid=} {loc=} '{strings[offs]}' {arg1=} {arg2=}")
```

Comment:
If the log-line has non-zero `loc-ID`, but we the env-var is OFF, then silently skip printing this loc-ID value.

Otherwise, on L96, invoke the LOC-decoding binary to get the file name / line# parts.